doc/sources/_README.md

```diff
@@ -86,9 +86,9 @@ because it lacks secure connections to protect your users' passwords.
 
 The system is installed and should be available for evaluation through
 
-* `http://localhost:5762/setup.pl`  
+* `http://localhost:5762/setup.pl`
```

Comment:
These trailing spaces were explicit, to force continuation of the content on the next line.

Reply:
Fixed in #6379